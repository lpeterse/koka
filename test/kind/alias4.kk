// test name hiding
import a = static/modA

alias aint  = a/aint
alias ok1   = aint
alias ok2   = a/aint
