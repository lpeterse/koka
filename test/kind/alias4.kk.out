kind/alias4/aint: V
kind/alias4/ok1: V
kind/alias4/ok2: V
public alias aint = a/aint = 2
public alias ok1 = aint = 3
public alias ok2 = a/aint = 2